packs/travisci/config.yaml

```diff
@@ -0,0 +1,7 @@
+git_auth_key: Your Git Authentication Key
```

Comment:
Also, it currently looks like `git_auth_key` configuration value is not used anywhere?